clang/include/clang/Driver/Options.td

```diff
@@ -4585,11 +4585,13 @@ let Flags = [TargetSpecific] in {
 def menable_experimental_extensions : Flag<["-"], "menable-experimental-extensions">, Group<m_Group>,
   HelpText<"Enable use of experimental RISC-V extensions.">;
 def mrvv_vector_bits_EQ : Joined<["-"], "mrvv-vector-bits=">, Group<m_Group>,
+  Visibility<[ClangOption, FlangOption]>,
   HelpText<"Specify the size in bits of an RVV vector register. Defaults to "
            "the vector length agnostic value of \"scalable\". Accepts power of "
            "2 values between 64 and 65536. Also accepts \"zvl\" "
-           "to use the value implied by -march/-mcpu. Value will be reflected "
-           "in __riscv_v_fixed_vlen preprocessor define (RISC-V only)">;
+           "to use the value implied by -march/-mcpu. On Clang, value will be "
+           "reflected in __riscv_v_fixed_vlen preprocessor define (RISC-V "
+           "only)">;
```

Comment:
I am very against using `HelpText` like this - it makes compiler "help" output (i.e. `flang-new -help`) unusable. Please split into `HelpText` and `DocBrief` and keep `HelpText` short and concise.